### ISSUE TITLE
Can't see all to the bed leveling wizard message.

### ISSUE THREAD

**Issue by a4jp-com** [daid/LegacyCura#1023]
After the connect to machine button is pressed a message comes up, but it says "Use the up/down buttons to move the bed and a"

What does the message say???

When we adjust the print nozzle, should it actually touch the bed or should we use a piece of paper?

Can you also add 0.1mm to adjust the height perfectly? 0.2 is nice but sometimes too much.

My printer need to be adjusted from the back two corners as well as front two. Is there a way to do this?

**Comment by daid** [comment]
Which OS?

**Comment by bebbi** [comment]
Mac OS 10.9.5 on macbook retina

**Comment by roowilliams** [comment]
+1 ... Cura 15.02.1 on Mac OS 10.9.5. What does it want me to do?!

**Comment by moritzmhmk** [comment]
+1 ... Mac OS 10.10 (btw. you can find the full messages here https://github.com/daid/Cura/blob/1fe02f44f02e93e628cf0307ed586eb8ae916fba/resources/locale/ko/LC_MESSAGES/Cura.po#L604)

**Comment by adeebshihadeh** [comment]
+1 on OS X 10.10.2

**Comment by Nandox7** [comment]
+1 on OSX 10.9.5 and 10.10

**Comment by rhyven** [comment]
Also happens under Linux Mint LMDE

**Comment by kleinlieu** [comment]
Getting this too on OS X 10.10 running the latest version.

**Comment by BagelOrb** [comment]
By latest version, do you mean 15.04, 15.06, 15.09.91 or 1.99 dev?

**Comment by kleinlieu** [comment]
Apologies, I had a hard time figuring out what the latest version was from the app's interface. I found the DMG file I downloaded though, and it's "Cura-15.02.1-MacOS". The "Check for Update" menu option said I was on the latest version though.

**Comment by lapentab** [comment]
Having the same issue but on Windows 10. Running version 15.04.4 which the updater says is latest.

**Comment by Minothor** [comment]
Having the same issue on:
Win10 (x64) - Cura 15.04.4
Tried to resize the message window but couldn't.
![cura message](https://cloud.githubusercontent.com/assets/2125437/14066465/d1041d22-f441-11e5-8438-5cbe18380e83.PNG)

**Comment by a4jp-com** [comment]
Windows 10 (64 bit)

**Comment by Ghostkeeper** [comment]
Latest version of daid's is 15.04.5, by the way, so that may be a bug in the updater too.
See https://ultimaker.com/en/products/cura-software for a download.

**Comment by daid** [comment]
Nope, this bug hasn't been fixed. It only happens if you run the bed leveling wizard on it's own, not as a new machine setup (for the Ultimaker Original)

**Comment by ahollenbach** [comment]
For anyone wondering what the full message is:

 **"Use the up/down buttons to move the bed and adjust your Z endstop."** ([link to source](https://github.com/daid/Cura/blob/b878f7dc28698d4d605a5fe8401f9c5a57a55367/Cura/gui/configWizard.py#L1316))
